### PR TITLE
Create Loading Screens for Matching

### PR DESCRIPTION
**Pull request by NCMoseley**

### CODE REVIEW

File: js/screens/MatchScreenLoading/MatchScreenLoadingContainer.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+import MatchScreenLoading from "./MatchScreenLoading";
+
+export default class MatchScreenLoadingContainer extends Component {
+  static navigationOptions = {
+    header: null
+  };
+  constructor() {
+    super();
+    this.state = {
```
Comment (siddharthparmar7) [inline js/screens/MatchScreenLoading/MatchScreenLoadingContainer.js]: Please use Redux store instead of the component state.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Thank you, Nate!

If you have any confusions, please ask me.

Thank you!

**Comment by NCMoseley** [comment]
Please review changes. Thank you very much.

File: js/screens/MatchScreenLoading/MatchScreenLoadingContainer.js (hunk 1 above)
Comment (siddharthparmar7) [inline js/screens/MatchScreenLoading/MatchScreenLoadingContainer.js]: Hi Nate,

Please use Redux store instead of the component state. Thank you.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the requested changes.

Thank you!

**Comment by NCMoseley** [comment]
Please review. Thanks

**Review by siddharthparmar7 — APPROVED** [review]
Thank you! Nate!

Good work.

**Pull request merged**